ultimaker_tough_pla_blue.xml.fdm_material

```diff
@@ -0,0 +1,140 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<fdmmaterial xmlns="http://www.ultimaker.com/material" version="1.3">
```

Comment:
[NITPICK] Isn't it version 1 for this particular material?

Reply:
@CoenSchalkwijk I think you mean the `<version>` in the body not here. 

I added a separate comment about this as well, shouldn't all the versions be `<version>1</version>` since these are new profiles?

Reply:
@atrebelo I was referring to the wrong version indeed. But would've probably missed it, if it wasn't visible in part of the diff so all the kudo's are yours.

Reply:
My Bad, wrong version to comment.